### ISSUE TITLE
How I train with a corpus format like this？

### ISSUE THREAD

**Issue by ScottishFold007** [UKPLab/sentence-transformers#240]
LCQMC is a problem semantic matching dataset constructed by Harbin Institute of Technology at COLING2018, the international summit of natural language processing, with the goal of determining whether the semantics of two problems are the same. The method of construction is to first extract high-frequency related questions from the Baidu Q&A for different fields, then conduct preliminary screening through Wasserstein distance, and finally annotate manually. The data set has 260068 pairs of annotated results, divided into three parts, 238766 training set, 8802 validation set and 12500 test set.

 Can I use this corpus for training？It's formatted like ：
![image](https://user-images.githubusercontent.com/36957508/82514118-65230a80-9b47-11ea-8d39-dde7046a76b4.png)

**Comment by nreimers** [comment]
I assume 1 mean similar, 0 means dissimilar?

You can use it with the same setup as with the sts training. In the STS example, scores are divided by 5 to normalize them to 0 ... 1. This would not be necessary here.

Best
Nils Reimers
